### PR TITLE
Updating Screensharing Policy

### PR DESCRIPTION
**Pull request by nej-nej**
Updating Screensharing Policy definition to improve clarity

### CODE REVIEW

File: Teams/meeting-policies-content-sharing.md [hunk 1]
```diff
@@ -46,22 +46,14 @@ You can also use the [Set-CsTeamsMeetingPolicy](/powershell/module/skype/set-cst
 
 ## Screen sharing mode
 
-This setting is a combination of a per-organizer and per-user policies. This setting controls whether desktop and window sharing is allowed in the user's meeting. Meeting participants who don't have any policies assigned (for example, external participants) inherit the policy of the meeting organizer.
+This setting is a per-user policy. This setting controls whether a user can share a desktop and window in a Teams meeting.
```
Comment (JohanFreelancer9) [inline Teams/meeting-policies-content-sharing.md:49]:
```suggestion
This setting is a per-user policy. The setting controls whether a user can share a desktop and window in a Teams meeting.
```

File: Teams/meeting-policies-content-sharing.md [hunk 2]
```diff
@@ -46,22 +46,14 @@ You can also use the [Set-CsTeamsMeetingPolicy](/powershell/module/skype/set-cst
...
 |**Not enabled**     |Screen sharing and application sharing turned off in the meeting.       |
 
-Let's look at the following example.
-
-|User |Meeting policy |Screen sharing mode |
-|---------|---------|---------|
-|Daniela  | Global   | Entire screen |
-|Amanda   | Location1MeetingPolicy  | Not enabled |
-
-Meetings hosted by Daniela allow meeting participants to share their entire screen or a specific application. If Amanda joins Daniela's meeting, Amanda can't share her screen or a specific application as her policy setting is **Not enabled**. In meetings hosted by Amanda, no one is allowed to share their screen or a single application, regardless of the screen sharing mode policy assigned to them.  Consequently, Daniela can't share her screen or a single application in Amanda's meetings.
```
Comment (JohanFreelancer9) [inline Teams/meeting-policies-content-sharing.md:57]:
```suggestion
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @nej-nej, kindly commit the suggested changes. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@MikePlumleyMSFT 
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**